<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmInspectProject</class>
 <widget class="QWidget" name="frmInspectProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_btn" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>1001</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QPushButton" name="btn_gpib_set">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>93</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>GPIB设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_gpib_disconnect">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>93</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>GPIB断开</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_inspect_mode">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>93</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>检定模式</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_debug_mode">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>93</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>调试模式</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_save">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>20</y>
      <width>93</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>保存数据</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_template_type">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>模板类别</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_assist_parameter">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_template_name">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>模板名称</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_template_type">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_read_method">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>210</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_read_method">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>210</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>读数方式</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_template_name">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_assist_parameter">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>辅助参数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_standard_source">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>120</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FLUKE   5500A    Calibrator</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_standard_source">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>180</y>
     <width>451</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_inpected_device">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>490</y>
     <width>451</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_standard_source_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>430</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>被检表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_read">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>650</y>
     <width>93</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>确定读数</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_begin_inspect">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>650</y>
     <width>93</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_inspect_result">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>521</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
